### ISSUE TITLE
how to disable install track

### ISSUE THREAD

**Issue by yangcheng** [mixpanel/mixpanel-android#754]
Hi, I am using a mixpanel for my app but I don't need to use installrefer for my case, however, if I remove 
`    implementation 'com.android.installreferrer:installreferrer:2.2'` from my build.gradle,  I got errors when I try to call mixpanel API to just capture event. Is there a way to use mixpanel API without adding installreferrer:installreferrer dependency ?

I am tried 5.9 and 6.0-beta-1, all have same issue.
Thanks 

```
Rejecting re-init on previously-failed class java.lang.Class<com.mixpanel.android.mpmetrics.InstallReferrerPlay>: java.lang.NoClassDefFoundError: Failed resolution of: Lcom/android/installreferrer/api/InstallReferrerStateListener;
    at com.mixpanel.android.mpmetrics.MixpanelAPI com.mixpanel.android.mpmetrics.MixpanelAPI.getInstance(android.content.Context, java.lang.String) (MixpanelAPI.java:253)
    at com.mixpanel.android.mpmetrics.MixpanelAPI ai.dongmi.barbell.manager.EventTrackingManager$mixpanel$2.invoke() (EventTrackingManager.kt:30)
    at java.lang.Object ai.dongmi.barbell.manager.EventTrackingManager$mixpanel$2.invoke() (EventTrackingManager.kt:29)
    at java.lang.Object kotlin.SynchronizedLazyImpl.getValue() (LazyJVM.kt:74)
    at com.mixpanel.android.mpmetrics.MixpanelAPI ai.dongmi.barbell.manager.EventTrackingManager.getMixpanel() (EventTrackingManager.kt:29)
```

**Comment by zihejia** [comment]
hi @yangcheng , thanks for bringing it up! Is there any concern not to include `implementation 'com.android.installreferrer:installreferrer:2.2'`? We are actually considering deprecating the install track feature and we'd like to get any feedback.

**Comment by zihejia** [comment]
Hi @yangcheng , we have released v6.0.0 which no longer contains this feature anymore so it should be good. I'm closing this one now.

**Comment by yangcheng** [comment]
thanks! ugprading now
